README.md

```diff
@@ -1,73 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```

Comment:
Kindly make sure to update the README file with more descriptive and project-related details.
You can use this [template](https://github.com/microverseinc/readme-template) from Microverse